base/common/ReplxxLineReader.cpp

```diff
@@ -81,6 +87,8 @@ ReplxxLineReader::ReplxxLineReader(
     /// it also binded to M-p/M-n).
     rx.bind_key(Replxx::KEY::meta('N'), [this](char32_t code) { return rx.invoke(Replxx::ACTION::COMPLETE_NEXT, code); });
     rx.bind_key(Replxx::KEY::meta('P'), [this](char32_t code) { return rx.invoke(Replxx::ACTION::COMPLETE_PREVIOUS, code); });
+
+    rx.bind_key(Replxx::KEY::meta('E'), [this](char32_t) { openEditor(); return Replxx::ACTION_RESULT::CONTINUE; });
```

Comment:
How about adding default `"\C-x\C-e"` (as readline has for `edit-and-execute-command`) ?

Reply:
Sounds reasonable. I wish we can have keybinds exported to some configuration file.